helpers/llm_helper.py

```
@@ -17,6 +46,14 @@
 try:
     import litellm
     from litellm import completion
+
+    # Ask LiteLLM to suppress debug information if possible
+    try:
+        litellm.suppress_debug_info = True
+    except Exception:
+        # Ignore if attribute is unavailable
+        pass
```

Ruff (0.14.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)